models/articles-model.js

```diff
@@ -31,4 +31,25 @@ exports.selectAllArticles = () => {
       const articles = result.rows;
       return articles;
     });
-};
\ No newline at end of file
+};
+
+exports.updateArticleVotes = (articleId, votes) => {
+  if (typeof articleId !== "number" || typeof votes !== "number") {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+  return exports
+    .selectArticles(articleId)
```

Comment:
I've already left feedback about this, I feel this could benefit from a custom checkExists function and a promise.all rather than reusing a model.

Reply:
<img width="698" alt="Screenshot 2024-08-27 at 14 18 17" src="https://github.com/user-attachments/assets/f44cdee5-c736-4888-9f06-01e7f7e40b8a">
<img width="631" alt="Screenshot 2024-08-27 at 14 18 26" src="https://github.com/user-attachments/assets/621b74df-32d8-44ac-b7c8-e4ac17d169fb">